www/js/onboarding/WelcomePage.tsx

```diff
@@ -100,6 +100,7 @@ const WelcomePage = () => {
       <Surface style={[onboardingStyles.page, { paddingVertical: 0 }]}>
         <View style={s.headerArea(windowWidth, colors)} aria-hidden={true} />
         <IconButton
+          accessibilityLabel="More info"
```

Comment:
this is not internationalized

Reply:
Resolved in https://github.com/e-mission/e-mission-phone/pull/1118/commits/cf7c86048930c54409a60e5eab09582ef4332cfc

Reply:
Associated commit https://github.com/e-mission/e-mission-translate/pull/53/commits/361fa6785d85050be3ecb04c649ff65baad81511 in PR https://github.com/e-mission/e-mission-translate/pull/53